### PR TITLE
chore(*): initial commit for workflow upgrade

### PR DESCRIPTION
**Pull request by kmala**

### CODE REVIEW

File: rootfs/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM quay.io/deis/base:0.3.0
```
Comment (mboersma) [inline rootfs/Dockerfile]: This should probably be up-to-date with tag `:0.3.4`.

File: DCO [hunk 2]
```diff
@@ -0,0 +1,36 @@
+Developer Certificate of Origin
```
Comment (helgi) [inline DCO:1]: Didn't we ditch this from other projects and keep it in the docs now?

Comment (kmala) [reply DCO:1]: i copied it from the controller repo.

File: rootfs/Dockerfile (hunk 1 above)
Comment (kmala) [reply rootfs/Dockerfile]: yes...changing it

File: README.md [hunk 3]
```diff
@@ -1 +1,23 @@
-# workflow-upgrade
\ No newline at end of file
+# workflow-upgrade
+[![Docker Repository on Quay](https://quay.io/repository/deisci/workflow-upgrade/status "Docker Repository on Quay")](https://quay.io/repository/deisci/registry-token-refresher)
```
Comment (vdice) [inline README.md]: This still points to a different project.

File: README.md [hunk 4]
```diff
@@ -1 +1,23 @@
...
+
+Deis (pronounced DAY-iss) Workflow is an open source Platform as a Service (PaaS) that adds a developer-friendly layer to any [Kubernetes](http://kubernetes.io) cluster, making it easy to deploy and manage applications on your own servers.
+
+For more information about the Deis Workflow, please visit the main project page at https://github.com/deis/workflow.
+
+We welcome your input! If you have feedback, please [submit an issue][issues]. If you'd like to participate in development, please read the "Development" section below and [submit a pull request][prs].
+
+# About
+The Workflow Upgrade service checks for the mismatch between the version of the daemonset pods and the [daemonset][daemons] and deletes the pods such that new pods come up with the latest changes. This is necessary until kubernetes has implemented the update strategy which is tracked through https://github.com/kubernetes/kubernetes/issues/22543.
```
Comment (vdice) [inline README.md:11]: Can we add a mention on how to use `workflow-upgrade`?

**Review by vdice — COMMENTED** [review]
In terms of CI, will we want to treat this project as a component similar to a given Workflow component?  (Meaning, pipelines for pr, master and release?)  Or, will Travis be the preferred route?

File: README.md (hunk 4 above)
Comment (kmala) [reply README.md:11]: Added the usage details.

File: README.md (hunk 3 above)
Comment (kmala) [reply README.md]: changed.

**Pull request merged**